packages/config/config/devices/0x027a/zse41.json

```diff
@@ -0,0 +1,111 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZSE41",
+	"description": "Open/Close XS Sensor",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xe001",
+			"zwaveAllianceId": 4276
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 3,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "On/Off",
+			"maxNodes": 5
+		}
+	},
```

Comment:
Would be interesting to know if we need this at all for the newer devices. Do you know which labels the device reports on its own?

Reply:
According to zwave-alliance, it just reports "Group 2" as the label for the second association group.

https://products.z-wavealliance.org/products/4280/assoc?noFilename=True

Reply:
Correct. It says Group 2, I was just trying to be more descriptive based on when it receives a report.

Reply:
Okay, then this is fine